Makefile.generated_files

```diff
@@ -286,6 +286,111 @@ $(DEEPCOPY_GEN):
 	touch $@
 
 #
+# Open-api generation
+#
+# Any package that wants open-api functions generated must include a
+# comment-tag in column 0 of one file of the form:
+#     // +openapi=<VALUE>
```

Comment:
so if this is +k8s:openapi then the `find` below is faster.  It's a minor point, but it is consistent with other "updated" tags we're using.  If/when we bust this out of the main repo, it's something we'd have to reconsider, but for now I suggest it.